### ISSUE TITLE
Check if typescript lib files are in `/lib`

### ISSUE THREAD

**Issue by drinchev** [smrq/tsify#67]
According to : https://github.com/Microsoft/TypeScript/pull/4039 , from now on TypeScript will have it's `d.ts` files ( such as `lib.d.ts, lib.es6.d.ts` ) in a different directory.

Since `tsify` looks for those files ( https://github.com/smrq/tsify/blob/v0.11.9/lib/Host.js#L19 ), support for `typescript@next` is broken.

Maybe something like this will work :

```
var libDefaultPath = languageVersion === ts.ScriptTarget.ES6 ?
			( require.resolve('typescript/bin/lib.es6.d.ts') || require.resolve('typescript/lib/lib.es6.d.ts') ) :
			( require.resolve('typescript/bin/lib.d.ts') || require.resolve('typescript/lib/lib.d.ts') );
```

Cheers.

**Comment by smrq** [comment]
Yeah, something like that should be fine.  Thanks for the heads up.

**Comment by smrq** [comment]
Fixed in 0.11.10
